.travis/metaSMT.sh

```diff
@@ -6,6 +6,7 @@ sudo apt-get -y install libboost1.55-dev libz3 libz3-dev libgmp-dev
 # Clone
 git clone git://github.com/hoangmle/metaSMT.git
 cd metaSMT
+git checkout 1f9e399
```

Comment:
It would be better but not yet necessary at the moment. I would be happier if we could do this later when metaSMT has a proper version. A FIXME has been added.